scvi/train/_trainingplans.py

```diff
@@ -661,6 +664,7 @@ def __init__(
         optim_kwargs: Optional[dict] = None,
         n_steps_kl_warmup: Union[int, None] = None,
         n_epochs_kl_warmup: Union[int, None] = 400,
+        scale_elbo: Union[float, None] = 1.0,
```

Comment:
Is this allowed to be None and if so what is the behavior? The docs make it seem like it requires a positive float.

Reply:
You are right - I copied this from another class where the default behaviour is to scale by the total number of cells.